### PR TITLE
Add webcam screen with toggleable displays

### PR DESCRIPTION
**Pull request by kylebrain**

### CODE REVIEW

**Review by michaelmartinelli15 — DISMISSED** [review]
Is togglable a word?

File: components/WebcamScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+      </Button>
+      {isDisplayed && <CheckBoxList {...props} />}
+    </View>
+  );
+};
+
+const CheckBoxList: React.FC<WebcamSelectionProps> = ({ webcamNames, webcamSelectionArray, updateWebcamSelection }: WebcamSelectionProps) => {
+
+
+  return (<Content>
```
Comment (ErylKenner) [inline components/WebcamScreen.tsx:32]: content goes on next line

File: components/WebcamScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+      return (
+        <ListItem key={index}>
+          <WebcamCheckbox 
+            updateWebcamSelection={(value: boolean) => { updateWebcamSelection(index, value); }}
+            initialCheckedState={webcamSelectionArray[index]}/>
+          <Text style={{ paddingLeft: 10 }}>{webcamNames[index]}</Text>
+        </ListItem>
+      );
+    })}
+  </Content>);
```
Comment (ErylKenner) [inline components/WebcamScreen.tsx:43]: ); goes on next line

File: components/WebcamScreen.tsx [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+          <View>
+            <Text style={{fontWeight : "bold"}}>{webcamArray[index].name}</Text>
+            {/* eslint-disable-next-line */}
+            <Image source={webcamArray[index].source} style = {{height: 200, width: 250, resizeMode : "stretch"}}/>
+          </View>
+          : <></>;
+      })}
+    </View>
+  );
+};
```
Comment (ErylKenner) [inline components/WebcamScreen.tsx:96]: newline

File: components/WebcamScreen.tsx [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+    newArray[index] = value;
+    setWebcamSelectionArray(newArray);
+  };
+
+  return (
+    <View>
+      <MultiSelectAccordion webcamSelectionArray={webcamSelectionArray} updateWebcamSelection={setWebcamSelection} webcamNames={webcamArray.map(webcam => webcam.name)} />
+      {webcamSelectionArray.map((value : boolean, index : number) =>
+      {
+        return value?
```
Comment (ErylKenner) [inline components/WebcamScreen.tsx:86]: Ternary? Perhaps an if-else block is better than a 8 line ternary statement

**Review by ErylKenner — CHANGES_REQUESTED** [review]

**Review by ErylKenner — APPROVED** [review]

**Review by michaelmartinelli15 — APPROVED** [review]

**Pull request merged**